src/main/java/mate/dao/CarDao.java

```diff
@@ -2,7 +2,10 @@
 
 import java.util.List;
 import mate.model.Car;
+import mate.model.Driver;
 
 public interface CarDao extends GenericDao<Car> {
     List<Car> getAllByDriver(Long driverId);
+
+    boolean addCarToDriver(Car car, Driver driver);
```

Comment:
we have already had method addDriverToCar in CarService, why do we need new one to CarDao?